internal/ansible/events/log_events.go

```diff
@@ -62,22 +64,52 @@ func (l loggingEventHandler) Handle(ident string, u *unstructured.Unstructured,
 		"job", ident,
 	)
 
+	// Parse verbosity from CR
+	verbosityAnnotation := 0
+	if annot, exists := u.UnstructuredContent()["metadata"].(map[string]interface{})["annotations"]; exists {
+		if verbosityField, present := annot.(map[string]interface{})["ansible.sdk.operatorframework.io/verbosity"]; present {
+			verbosityValue, err := strconv.Atoi(verbosityField.(string))
+			if err != nil {
+				logger.Error(err, "Unable to parse verbosity value from CR.")
+			} else {
+				verbosityAnnotation = verbosityValue
+			}
+		}
```

Comment:
```suggestion
	verbosityAnnotation := 0
	if annot, exists := u.UnstructuredContent()["metadata"].(map[string]interface{})["annotations"]; exists {
		if verbosityField, present := annot.(map[string]interface{})["ansible.sdk.operatorframework.io/verbosity"]; present {
			verbosityAnnotation, err := strconv.Atoi(verbosityField.(string))
			if err != nil {
				logger.Error(err, "Unable to parse verbosity value from CR.")
			}
		}
```

Reply:
Hi @jmrodri, making those changes results in the following errors during the building of the binary:
``internal/ansible/events/log_events.go:71:4: verbosityAnnotation declared but not used``
``internal/ansible/events/log_events.go:82:3: verbosityEnvVar declared but not used``
``make: *** [install] Error 2``

I believe it's because of the redeclaration of `verbosityAnnotation` in a different scope (Line 68 and Line 71).
Perhaps I could change line 71 to an assignment instead of a declaration. The same would follow for `verbosityEnvVar`.